php-practice/section2.php

```diff
@@ -117,6 +139,18 @@ function printNewline() {
 // HINT: Use a foreach loop
 // =======================================================================
 
+$sumArray = array(1, 2, 3, 4); 
+
+function ($sumArray) {
+	foreach ($sumArray as $value);
+	$total = 0;
+	$total += $value;
+	return ($total);
+};
+
+print_r($sumArray);
```

Comment:
So here we will print the output value of the function. But remember, the function expects an array to be passed in. So like this:

```
echo sumArray($myArray);
```

Assuming the array was named $myArray and the function was named sumArray.